VAMobile/src/screens/HealthScreen/Appointments/UpcomingAppointments/UpcomingAppointmentDetails.tsx

```diff
@@ -303,7 +304,11 @@ const UpcomingAppointmentDetails: FC<UpcomingAppointmentDetailsProps> = ({ route
 
           <Atlas_AppointmentData />
           <SpecialInstructions />
-
+          {featureEnabled('patientCheckIn') && (
+            <Box my={theme.dimensions.gutter} accessibilityRole="button" mr={theme.dimensions.buttonPadding}>
```

Comment:
Looks like VAButton already has `accessibilityRole="button"` so I think it can be removed here:
```suggestion
            <Box my={theme.dimensions.gutter} mr={theme.dimensions.buttonPadding}>
```